### DISCUSSION TITLE
What is the recommended number of cores/threads to dedicate to this container?

### DISCUSSION THREAD

**Discussion by marktheawesome** [wolveix/satisfactory-server#367] [Q&A] [answered]
Is this container setup to be multi threaded or just single threaded?  If it is multi threaded how many threads/cores do you recommend  using if running on Unriad?

That you.

**Comment by wolveix** [comment, marked as answer]
Hey @marktheawesome, the container is a lightweight wrapper for the game, and doesn't impose anything itself. The game server is a typical game server, and therefore requires strong single-core performance (passmark of 3k plus in my experience).

**Reply by marktheawesome** [reply]
Thank you!
